solution02/src/test/resources/user.properties

```diff
@@ -0,0 +1,67 @@
+##  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+#  WSO2 Inc. licenses this file to you under the Apache License,
+#  Version 2.0 (the "License"); you may not use this file except
+#  in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+
+#  under the License.
+#
+
+#server
+server_host=localhost
+server_port=9443
+
+# Tomcat
+tomcat_host=localhost
+tomcat_port=8080
+
+# User Management
+admin_username=admin
+admin_password=admin
+
+# IDP
+FbIdentityProviderName=FacebookIdpsolution2
+FbIdentityProvider_description=solution2FacebookIDP1
+GoogleIdentityProviderName=solution2GoogleIDP1
+GoogleIdentityProvider_description=solution2GoogleIDP1
+
+# SP
+spName=travelocity.com
+spDescription=travelocity
+travelocityIssuer=travelocity.com
+GoogleSPName=travelocity.com
+GtravelocityIssuer=travelocity.com
+
+
+#Facebook
+Fusername=testuserFacebook
+Fpassword=facebook123
+role=roleFacebook
+permission=/permission/admin
+FbClientId=214847805723276
+FbSecret=47d454e7798a92ce71588148b90497eb
+FbCallbackUrl=commonauth
+FbScope=email
+FbUserInfoFields=id,name,gender,email,first_name,last_name,age_range,link
+
+#Google
+Gusername=testuserGoogle
+Gpassword=google123
+Grole=roleGoogle
+Gpermission=/permission/admin
+Alias=oauth2/token
+GoogleClientId=697121079413-c0sntgirfgicfcaacljq8bd8nlan9qvs.apps.googleusercontent.com
+GoogleSecret=A_bwPSmYOQcwvdS1qT3qVE8Y
```

Comment:
same as comments in https://github.com/wso2-incubator/identity-test-integration/pull/133#pullrequestreview-84032116